tricycle:  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Position Controllers ---------------------------------------
  joint0_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: base_wheel_0_joint
    pid: {p: 1, i: 0.2, d: 0}
  joint1_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: base_wheel_1_joint
    pid: {p: 1, i: 0.2, d: 0}
  joint2_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: base_wheel_2_joint
    pid: {p: 1, i: 0.2, d: 0}


  joint0_name: base_wheel_0_joint
  joint1_name: base_wheel_1_joint
  joint2_name: base_wheel_2_joint
